src/tests/test_zksync_cli.sh

```diff
@@ -99,6 +99,54 @@ verify_result
 
 ###
 
+echo "Test #1869"
+echo "---------------------------------"
+
+echo "> zksync-cli wallet balance on Sepolia Testnet"
+npx zksync-cli wallet balance --chain zksync-sepolia --address 0x52B6d10d7d865B3d4103f8809AA3521288568f46
+
+echo "> zksync-cli wallet balance on Zksync Mainnet"
+npx zksync-cli wallet balance --chain zksync-mainnet --address 0x52B6d10d7d865B3d4103f8809AA3521288568f46
+
+echo "> zksync-cli wallet balance on Goerli Testnet"
+npx zksync-cli wallet balance --chain zksync-goerli --address 0x52B6d10d7d865B3d4103f8809AA3521288568f46
+
+
+verify_result
+
+###
+
+echo "Test #1718"
+echo "---------------------------------"
+
+echo "> zksync-cli dev update <module>"
+npx zksync-cli dev update zkcli-portal
+
+verify_result
+
+###
+
+echo "Test #1874"
+echo "---------------------------------"
+
+echo "> zksync-cli contract read"
+npx zksync-cli contract read --chain zksync-sepolia --contract 0xE6c391927f0B42d82229fd3CFe3426F209D16b48 --method "greet() view returns (string)" --output string
+
+verify_result
+
+###
+
+echo "Test #1875"
+echo "---------------------------------"
+
+echo "> zksync-cli contract write"
+npx zksync-cli contract write --chain zksync-sepolia --contract 0xE6c391927f0B42d82229fd3CFe3426F209D16b48 --method "setGreeting(string _greeting) " --args "New Test ARG" --private-key 32e2e997e1a2d91cee03f77f903103ce6f50301e125307cc4bcaa87313f1a13e
```

Comment:
we must not store --private-key in direct access even if it belongs to a testnet. 
I will replace it to the test solution with a fix:   testnet_private_key: process.env.E2E_TESTNET_PK

Reply:
Thank you! Of course make sense from security side.